tests/framework/Optimizers/SimulatedAnnealing/BoltzmanSA/gold/opt_export.csv

```diff
@@ -0,0 +1,2 @@
+trajID,filename
```

Comment:
As with the XML file, I don't think this header CSV is worth golding; there's no unique information here. I would just gold the "*_0.csv" file, for the sake of streamlining the repository.

Reply:
Makes sense. Thanks